pkg/httputil/tracing.go

```diff
@@ -137,7 +137,6 @@ func TracingMiddleware(requestIDHeaderName string, fields logging.Fields, traceR
 				"took":             time.Since(startTime),
 				"status_code":      responseWriter.StatusCode,
 				"sent_bytes":       responseWriter.ResponseSize,
-				"request_body":     requestBodyTracer.bodyRecorder.Buffer,
```

Comment:
We are we removing the request body?

can we remove line 130-132
```
// record request body as well
			requestBodyTracer := newRequestBodyTracer(r.Body, RequestTracingMaxRequestBodySize)
			r.Body = requestBodyTracer
```

Reply:
You were actually right that I should never have removed the request.  Restored that, now logs are glorious!